test/ctia/task/rollover_test.clj

```diff
@@ -64,7 +66,7 @@
            (is (= 3 (count-indices)))))))))
 
 (deftest rollover-stores-error-test
-  (with-redefs [es-index/rollover! (fn [_ alias _]
+  (with-redefs [es-index/rollover! (fn [_ alias _ _]
```

Comment:
Minor: perhaps also provide a 3 arg arity?

Reply:
I finally switched back to the 3 args arity because the 4th parameter was empty (back and forth with ductile changes, sorry). In any case this refined function is a ductile function which is used with only one arity (finally 3 args ;-)) in the rollover task.